src/components/CardList/ReactionCardList.js

```diff
@@ -0,0 +1,107 @@
+import { useState, useEffect } from 'react';
+import { ReactComponent as NextIcon } from 'assets/icons/arrow_right.svg';
+import styled from 'styled-components';
+import Card from 'components/CardList/Card';
+import useRequest from 'hooks/useRequest';
+
+function ReactionCardList({ title }) {
+  const [page, setPage] = useState(0);
+  const limit = 4;
+  const { data, isLoading, error } = useRequest({
+    url: `1-5/recipients/`,
+    method: 'get',
+    params: { limit: 1000, sort: 'like' },
+  });
+
+  const [sortedData, setSortedData] = useState([]);
+
+  useEffect(() => {
+    if (data && data.results) {
```

Comment:
이부분 옵셔널 체이닝 (?.) 말고 && 연산자 사용하신 이유가 있을까요 ? 옵셔널 체이닝이 코드길이가 짧아져서 가독성이 좋아진다고 하더라구요 ! 
```suggestion
    if (data?.results) {
```

Reply:
앗 그렇군요!! 알려주셔서 감사합니다👀👀☺️☺️